src/tv2-common/content/dve.ts

```diff
@@ -254,16 +252,14 @@ export function MakeContentDVE2<
 			} else if (prop?.match(/DEFAULT/)) {
 				boxMap[targetBox - 1] = { source: `DEFAULT SOURCE` }
 			} else if (prop) {
-				if (videoId && !usedServer) {
+				if (videoId) {
```

Comment:
All this nesting makes me sad... :( 
The body of the two conditionals for videoId are now duplicated. (Not sure if you should do something about it now, this needs of a big refactor)

Reply:
Agree. In this round I just removed dead code. If I lack tasks to do, I'll have a look at it 😄